modules/govuk/manifests/deploy/config.pp

```diff
@@ -127,6 +127,12 @@
     'GOVUK_CSP_REPORT_URI': value      => $csp_report_uri;
   }
 
+  if $::aws_migration and ($::aws_environment != 'production') {
+    govuk_envvar {
+      'GOVUK_DATA_SYNC_PERIOD': value => data_sync_times('time_range');
```

Comment:
So this does seem to be setting it (don't worry about the lack of leading zeros - `Time.parse` still parses them fine):

```bash
$ more /etc/govuk/env.d/GOVUK_DATA_SYNC_PERIOD
22:0-8:0
```

But if I `echo $GOVUK_DATA_SYNC_PERIOD` I get nothing. Do we know how/whether these variables get exported as traditional ENV vars @kevindew ?

Reply:
I'm not sure exactly how they work - I think the govuk_setenv command sets them:

```
kevindew@ec2-integration-blue-email_alert_api-ip-10-1-6-234:~$ govuk_setenv email-alert-api env | grep GOVUK_DATA_SYNC_PERIOD
GOVUK_DATA_SYNC_PERIOD=22:0-8:0
```

You can see the env var if you go into an app console. For example Email Alert API on integration: 

```
irb(main):002:0> ENV["GOVUK_DATA_SYNC_PERIOD"]
=> "22:0-8:0"
```

Reply:
That's a relief - thanks for confirming!